packages/react/carbon.yml

```diff
@@ -0,0 +1,699 @@
+      framework: react
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-errorboundary
+      # TODO: @Aubrey - double check tags
+      tags:
+        - system-feedback
+    file-uploader:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/file-uploader/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-fileuploader--file-uploader
+    flex-grid:
+      name: Flex grid
+      framework: react
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-flexgrid
+      # TODO: @Aubrey - double check tags
+      tags:
+        - shell
+    form:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/form/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-form
+    form-group:
+        name: Form group
+        framework: react
+        status: stable
+        type: component
+        platform: web
+        # TODO: add thumbnail
+        thumbnailPath: '' 
+        demoLinks:
+          - type: storybook
+            name: Storybook
+            action: link
+            url: https://react.carbondesignsystem.com/?path=/docs/components-formgroup
+        # TODO: @Aubrey - double check tags
+        tags:
+          - form
+          - input-control  
+    form-label:
+        name: Form label
+        framework: react
+        status: stable
+        type: component
+        platform: web
+        # TODO: add thumbnail
+        thumbnailPath: '' 
+        demoLinks:
+          - type: storybook
+            name: Storybook
+            action: link
+            url: https://react.carbondesignsystem.com/?path=/docs/components-formlabel
+        # TODO: @Aubrey - double check tags
+        tags:
+          - content-element
+    icon-button:
+        name: Icon button
+        framework: react
+        status: stable
+        type: component
+        platform: web
+        # TODO: add thumbnail
+        thumbnailPath: '' 
+        demoLinks:
+          - type: storybook
+            name: Storybook
+            action: link
+            url: https://react.carbondesignsystem.com/?path=/docs/components-iconbutton
+        # TODO: @Aubrey - double check tags
+        tags:
+          - content-element
```

Comment:
Keep input-control. Add media tag. Delete content-element. Content element is a grey area but think of it more like headers, paragraphs, dividers whereas a button has an interaction and nicely fits into input-control.